src/Component/tests/spec/State/ProviderSpec.php

```diff
@@ -73,7 +73,7 @@ function it_throws_an_exception_when_configured_provider_is_not_a_provider_insta
         $locator->has('\stdClass')->willReturn(true);
         $locator->get('\stdClass')->willReturn(new \stdClass());
 
-        $this->shouldThrow(new \InvalidArgumentException('Expected an instance of Sylius\Component\Resource\State\ProviderInterface. Got: stdClass'))
+        $this->shouldThrow(new \InvalidArgumentException('Expected an instance of Sylius\Resource\State\ProviderInterface. Got: stdClass'))
```

Comment:
As idea
```suggestion
        $this->shouldThrow(new \InvalidArgumentException(sprintf('Expected an instance of %s. Got: %s', ProviderInterface::class, \stdClass::class)))
```

Reply:
Yeah, I think I've read that it's better not to use constants and things like that in specs/tests.
@lchrusciel @Zales0123?

Reply:
I don't have a strong opinion about that. Will the test still make sense if we would move provider to other namespace? Yup. Both approaches are fine for me